modules/cnf-deploying-the-numa-aware-scheduler.adoc

```diff
@@ -88,16 +31,7 @@ spec:
 $ oc create -f nro-scheduler.yaml
 ----
 
-.Verification
-
-. Verify that the performance profile was applied by running the following command:
-+
-[source,terminal]
-----
-$ oc describe performanceprofile <performance-profile-name>
-----
-
-. Verify that the required resources deployed successfully by running the following command:
+. After a few minutes run the following command to verify that the required resources deployed successfully:
```

Comment:
applying the scheduler doesn't trigger reboot so no need to wait few mins, it is just matter of secs